app/src/org/commcare/android/view/GridEntityView.java

```diff
@@ -92,7 +96,7 @@
 	 * @param detail
 	 * @param entity
 	 */
-	public GridEntityView(Context context, Detail detail, Entity entity, AudioController controller) {
+	public GridEntityView(@SuppressWarnings("LocalCanBeFinal") Context context, @SuppressWarnings("LocalCanBeFinal") Detail detail, @SuppressWarnings("LocalCanBeFinal") Entity entity, @SuppressWarnings("LocalCanBeFinal") AudioController controller) {
```

Comment:
Why do you have all these SuppressWarnings annotations? 
I think the add more clutter than value, but I'm saying that without really understanding why they are there.